rever/activities/push_tag.xsh

```diff
@@ -0,0 +1,59 @@
+"""Activity for pushing remote tags."""
+import re
+
+from rever import vcsutils
+from rever.activity import Activity
+from rever.tools import eval_version
+
+
+class PushTag(Activity):
+    """Pushes the current tag up to a remote repository.
+
+    This activity takes the following parameters:
+
+    :$PUSH_TAG_REMOTE: str or None, remote URL to push tags to.
+    :$PUSH_TAG_TARGET: str or None, remote branch to push to once the tag has been made.
+        The default is None, which uses the current branch.
+    :$GITHUB_ORG str or None, GitHub org to push to if no $TAG_REMOTE
+    :$GITHUB_REPO = str or None GitHub repo to push to if no $TAG_REMOTE
+    """
+
+    def __init__(self, *, deps=frozenset(('tag', ))):
+        super().__init__(name='push_tag', deps=deps, func=self._func,
+                         desc="Tags the current version.")
+
+    def _func(self, remote=None, target=None, org=None, repo=None):
+        if remote is None:
+            # Pull from the org and repo
+            if org and repo:
+                remote = 'git@github.com:{org}/{repo}.git'.format(org=org,
+                                                                  repo=repo)
```

Comment:
These don't actually use the GITHUB_ORG and GITHUB_REPO vars.  Also, it seems like a double effort to have both PUSH_TAG_REPO and GITHUB_REPO.

Reply:
Also, also you should only set the remote to be a git hub URL if the [$REVER_VCS](https://regro.github.io/rever-docs/envvars.html#rever-vcs) is git